events.go

```diff
@@ -98,3 +99,13 @@ information is displayed once every 5 seconds.`,
 		group.Wait()
 	},
 }
+
+func containerStatus(container libcontainer.Container) {
```

Comment:
I don't think it makes sense to have a function that calls `fatal()`, since if you just read the calling code it's not clear the function might kill the whole process. Do it in the caller.